src/main/java/com/powsybl/openloadflow/network/VoltageControl.java

```diff
@@ -40,10 +42,11 @@ public enum MergeStatus {
 
     protected boolean disabled = false;
 
-    protected VoltageControl(double targetValue, Type type, int priority, LfBus controlledBus) {
+    protected VoltageControl(double targetValue, Type type, int targetPriority, LfBus controlledBus) {
         super(targetValue);
         this.type = Objects.requireNonNull(type);
-        this.priority = priority;
+        this.priority = LfNetworkParameters.VOLTAGE_CONTROL_PRIORITIES_DEFAULT_VALUE.indexOf(type.name()); // Align default activation priorities with default target priorities
+        this.targetPriority = targetPriority;
```

Comment:
@annetill, I change here to align default values of "priority" and "targetPriority". Now a modification of the hard coded voltage control activation priorities will be applied to the default voltage target priorities.

Reply:
further cleaned in b8b76ad4cb102e5acc3fc8ff78e074ec613cb8dc
resolving